CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# This file (c) 2017 AlertAvert.com.  All rights reserved.
+
+project(SimpleHttpRequest)
+cmake_minimum_required(VERSION 3.4)
+
+# Version number
+set(RELEASE_MAJOR 0)
+set(RELEASE_MINOR 2)
+set(RELEASE_PATCH 0)
```

Comment:
let's make same as

#define SIMPLE_HTTP_REQUEST_VERSION_MAJOR 0
#define SIMPLE_HTTP_REQUEST_VERSION_MINOR 1
#define SIMPLE_HTTP_REQUEST_VERSION_PATCH 0

Reply:
Ok - updated that, but the reason was that I saw you'd already released `0.1.0`, so my assumption was that any subsequent build would be on a newer version.

Reply:
Oh, bump the version(with changing the SIMPLE_HTTP_REQUEST_VERSION_... macros) as long as it reach 1.0.0 :)